app/views/users/show.html.erb

```diff
@@ -1 +1,66 @@
-<p>Sigle user goes here</p>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3">
+    <div class="card-title d-flex justify-content-between p-2">
+      <h6 class="">Username: <%= @user.name %></h6>
+      <p class="mx-4" style="position: relative">
+        Posts:
+        <span
+          style="
+            position: absolute;
+            width: 20px;
+            height: 20px;
+            border-radius: 50%;
+            background: blue;
+            color: white;
+            font-size: 12px;
+            text-align: center;
+          "
+          class=""
+          ><%= @user.posts_counter %></span
+        >
+      </p>
+    </div>
+    <div class="card-body">
+      <p class="card-text">Bio: <%= @user.bio %></p>
+      <div class="links">
+        <%= link_to 'Posts', user_posts_path(@user), class: 'btn-sm btn
+        btn-info' %> <%= link_to 'Back', root_path(@user), class: 'btn-sm btn
+        btn-default' %>
+      </div>
+    </div>
+    <section class="py-1 container">
+      <h6>Posts:</h6>
+      <div class="post">
+        <% @user.posts.each do |post| %>
+        <div class="d-flex justify-content-between bg-light p-1 rounded mb-1">
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Kindly use the `fetch_recent_posts` method in your `user` model to render the 3 most recent posts only.